src/app/compressed-air-assessment/system-profile-graphs/system-profile-graphs.component.html

```diff
@@ -1,21 +1,21 @@
 <div class="print-container">
     <div class="print-avoid-break">
-        <label class="group-label bg-white ml-2">{{labelName}} System Capacity</label>
+        <label class="group-label bg-white">{{labelName}} System Capacity</label>
```

Comment:
The label text is against the border of the div and should also get a pl 2 or 3.

Reply:
good catch!